rfcs/0000-User&HostIntegrationRFC.md

```diff
@@ -0,0 +1,254 @@
+# 0000: Asset Integration
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+This proposal aims to extend existing ECS fieldset to store inventory metadata for hosts, users from external application repositories. Using ECS to store such fields will improve metadata queryring and retrieval across various use cases.
+
+Within the Security `Entity Analytics` initiatve, we refer to hosts and users as `entities`. We also refer to hosts/ users as `assets` across generic security or obserbality use cases. For simplificaiton, entities, assets will refer to host and user objects.
+
+This proposal includes the following:
+* Additional fields in the `users` object 
+* Introduces a new field set called `assets`
+<!-- * Additional fields in the `host` object --->
+
+This proposal will also facilitate storing of host and user inventory within security solution (aka. the entity store).
+
+### References:
+* [Entity Analytics Plugin Architecture for Host & User Integrations](https://github.com/elastic/beats/issues/33866#issuecomment-1165709940)
+* 0038: Extend event.kind allowed values with asset [Existing RFC](https://github.com/elastic/ecs/blob/main/rfcs/text/0038-event-kind-asset.md#0038-extend-eventkind-allowed-values-with-asset)
+* [Creation of an asset `super-schema`](https://github.com/elastic/security-team/issues/2347)
```

Comment:
Please remove links to non-public repos since this proposal will be public facing. If you want to link the issues together, comment on the private issue referencing the public one.